test/functional/news_letter_controller_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# TODO: We had some functional tests that tested CRUD (create-update-delete)
```

Comment:
did the todo get resolved?  if yes, should we remove the comment and rely on the doc string.  if no, what is the remaining todo functionality?